### PR TITLE
Allow using jest-prop-type-error with Create React App

### PR DESCRIPTION
**Pull request by YPCrumble**
Hi - thanks so much for this package! Very useful.

Here's a PR to allow easy integration with Create React App. Let me know what you think or if you have any suggested changes?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,3 +7,5 @@ console.error = (message, ...args) => {
 
   error.apply(console, [message, ...args]);
 };
+
+module.exports = console.error;
```
Comment (eliihen) [inline index.js]: This line is actually not needed. The module mutates the global console object, and thus does not need to expose anything.
I double-checked this in create-react-app, and it works fine without any changes.
Take out this line and I will merge the change to the README :slightly_smiling_face:

**Review by eliihen — CHANGES_REQUESTED** [review]
Thanks for this PR!

I will absolutely merge something which makes this library easier to use for `create-react-app` users. The documentation is great, but I have a comment on the code change. Have a look at that and we can merge this!

File: index.js (hunk 1 above)
Comment (YPCrumble) [reply index.js]: @esphen you're absolutely right, thanks for the review!

**Comment by YPCrumble** [comment]
@esphen just updated my PR :). Thanks!

**Review by eliihen — APPROVED** [review]

**Comment by eliihen** [comment]
Thanks again for the PR!

**Pull request merged**